lib/dor/services/client/object_version.rb

```diff
@@ -27,12 +27,11 @@ def current
         end
 
         # Determines if a new version can be opened for a DOR object.
-        # @param params [Hash] optional params (see dor-services-app)
         # @raise [NotFoundResponse] when the response is a 404 (object not found)
         # @raise [UnexpectedResponse] when the response is not successful.
         # @return [Boolean] true if a new version can be opened
         # rubocop:disable Metrics/MethodLength
-        def openable?(**params)
```

Comment:
openable no longer takes params, as they were never used.

Reply:
Since we're changing the API here, should this require a new major version to be released?

Reply:
I looked at all our code using github search, and no app was using params anywhere.  Happy to bump to a major version after this PR if you feel it's warranted.